docs/definitions/testing_split.md

```diff
@@ -0,0 +1,7 @@
+# `ml:testing_split`
+
+Other known names: `test`.
+
+The testing split is a data set used to provide an unbiased evaluation of a final model fit on the training data set. This data split might be hidden by the dataset author.
```

Comment:
nit: should maybe be "data set author" instead of "dataset author" for consistency?